### ISSUE TITLE
Add search "ingvarr"

### ISSUE THREAD

**Issue by torerobo** [sherlock-project/sherlock#480]
```
  "ingvarr.net.ru": {
    "errorMsg": "Пользователь не найден",
    "errorType": "message",
    "rank": 165229,
    "url": "http://ingvarr.net.ru/index/8-0-{}",
    "urlMain": "https://http://ingvarr.net.ru/",
    "username_claimed": "red",
    "username_unclaimed": "noonewouldeverusethis7"
  },

```

**Comment by hoadlck** [comment]
The "urlMain" attribute is not correct.

**Comment by torerobo** [comment]
> Атрибут "urlMain" неверен.

Sorry!

On
![2](https://user-images.githubusercontent.com/35532898/71626047-376f1080-2bfc-11ea-8674-420bf439b066.png)

```
  "ingvarr.net.ru": {
    "errorMsg": "Пользователь не найден",
    "errorType": "message",
    "rank": 165229,
    "url": "http://ingvarr.net.ru/index/8-0-{}",
    "urlMain": "http://ingvarr.net.ru/",
    "username_claimed": "red",
    "username_unclaimed": "noonewouldeverusethis7"
  },
```

**Comment by hoadlck** [comment]
Merged in #500.
